src/gd_kotlin.cpp

```diff
@@ -430,12 +445,14 @@ void GDKotlin::register_classes(jni::Env& p_env, jni::JObjectArray p_classes) {
     LOG_INFO("Loading classes ...")
 #endif
     for (auto i = 0; i < p_classes.length(p_env); i++) {
-        auto kt_class = new KtClass(p_classes.get(p_env, i), class_loader);
+        jni::JObject clazz = p_classes.get(p_env, i);
+        auto kt_class = new KtClass(clazz, class_loader);
```

Comment:
This was something we did not catch. Can you add pointer to auto, so that is it explicit.